code/spell.lisp

```diff
@@ -1,135 +0,0 @@
-(in-package #:spell)
```

Comment:
Just noticed that `spell/simple` is broken, because it still refers to this file.

Reply:
No longer an issue once `spell/simple` is an alias for `spell`.